index.html

```diff
@@ -375,7 +375,10 @@ <h3>
       a <dfn>[[\PenaltyDuration]]</dfn> integer set as part of the [=reset observation window=] steps.
     </li>
     <li>
-      a <dfn>[[\ChangesCountMap]]</dfn> integer set as part of the [=reset observation window=] steps.
+      a <dfn>[[\ChangesCountMap]]</dfn> [=ordered map=], [=map/keyed=] on a {{PressureSource}},
+      representing the [=source type=] to which the last state change belongs.
```

Comment:
Would this proposed alternative wording mean the same:

`representing the [=source type=] that triggered transition to the [=current pressure state=].`

If that makes sense, then on L385 how about this wording:

`representing the [=source type=] of the last {{PressureRecord}}.`

(Disclaimer: I read this PR without additional context just trying make those lines read more like English, so this may not be what you try to specify.)

Reply:
ok, Thanks I ll make the change!